### ISSUE TITLE
[Rule change] Add threshold for prefer-moving-to-variable

### ISSUE THREAD

**Issue by Pomis** [dart-code-checker/dart-code-metrics#815]
**What rule do you want to change?**
prefer-moving-to-variable

**How will the change be implemented? (New option, new default behavior, etc.)?**
Make it a metric with adjustable number of repeats to trigger the lint
```
metrics:
  - allowed-duplicated-chains: 2
```
**Please provide some example code that this change will affect:**
Now this fragment gets lint warning:
```dart
      BottomSheetSelector(
        ...
        title: 'ss_specialties'.tr(), // lint
        searchHint: 'ss_specialties'.tr(), // lint
        ...
      ),
```
However, I don't want to introduce a new variable just for this. Would be nice to allow adjusting the amount of repeats.

**What does the rule currently do for this code?**
Triggers lint
**What will the rule do after it's changed?**
Ignore this
**Are you willing to submit a pull request to implement this change?**
No

**Comment by Cilestal** [comment]
Is there any workaround to solve this problem?

**Comment by incendial** [comment]
I think the only workaround is to use `// ignore` comments

**Comment by incendial** [comment]
Available in 4.17.0 🚀
